README.md

```diff
@@ -0,0 +1,187 @@
+
+```
+git clone git@github.com:ansible/awx-resource-operator.git
+```
+
+2. Log in to your cluster.
+
+```
+kubectl login <cluster-url>
+```
+
+3. Run the make target
+
+```
+NAMESPACE=awx IMG=quay.io/ansible/awx-resource-operator:latest make deploy
+```
+
+This will create a deployment named `resource-operator-controller-manager` in the namespace specified, in this case the `awx` namespace. The resulting operator pod will reconcile any AnsibleJob or JotTemplate resources created.
+
+
+## Usage
+
+Now that the awx-resource-operator is installed in your cluster, you can try creating a k8s resource to manage your AWX instance.
+
+First, you need create a k8s secret with connection information for your AWX instance though. Below is example yaml that you can save to a file, edit, and apply to your cluster.
+
+### Create an AWX Access Token
+
+Create an OAuth2 token for your user in the AWX UI.
+
+1. Navigate to Users
+2. Select the username you wish to create a token for
+3. Click on tokens, then the green plus icon
+4. Application can be left empty, input a description and select the read/write scope.
+
+> Alternatively, you can create one at the command-line using the `create_oauth2_token` manage command ([docs](https://docs.ansible.com/automation-controller/latest/html/administration/tower-manage.html#create-oauth2-token))
+
+### Create a Connection Secret
+
+To make this connection information available to the operator, create a k8s secret with the host and token value. Here is example yaml for the connection k8s secret.
+
+```
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: awx-access
+  type: Opaque
+stringData:
+  token: <generated-token>
+  host: https://my-awx-host.example.com/
+```
+
+Save this to a file, for example, `awx-connection-secret.yml`, edit it with your host and token value, the apply it to your cluster.
+
+```
+kubectl create -f awx-connection-secret.yml
+```
+
+### AnsibleJob
+
+Launch an automation job on AWX by creating an AnsibleJob resource. Be sure to specify the connection secret on `tower_auth_secret` as well as the Job Template you wish to launch.
+
+```
+---
+apiVersion: tower.ansible.com/v1alpha1
+kind: AnsibleJob
+metadata:
+  generateName: demo-job-1 # generate a unique suffix per 'kubectl create'
+spec:
+  tower_auth_secret: awx-access
+  job_template_name: Demo Job Template
+```
+
+It is possible to configure other things like inventory, extra vars, and time to live for the job.
+
+```
+spec:
+  tower_auth_secret: awx-access
+  job_template_name: Demo Job Template
+  inventory: Demo Inventory                # Inventory prompt on launch needs to be enabled
+  runner_image: quay.io/ansible/awx-resource-runner
+  runner_version: latest
+  job_ttl: 100
+
+  extra_vars:                              # Extra variables prompt on launch needs to be enabled
+     test_var: test
+```
+
+Note that prompt on launch needs to be enabled for inventories and extra vars if you are specifying those.
+
+
+### JobTemplate
```

Comment:
Curious do we not have this functionality for Workflows?

Reply:
We do.  I should document that too.